packages/@aws-cdk/cfnspec/build-tools/create-missing-libraries.ts

```diff
@@ -107,6 +107,7 @@ async function main() {
       types: 'lib/index.d.ts',
       jsii: {
         outdir: 'dist',
+        projectReferences: true,
```

Comment:
This seems like opt in to me

Reply:
If it wasn't for the change in `tools/cdk-build-tools/lib/package-info.ts`, where it was *always* on, and which is the whole point of this PR.